301-azure-relay-create-authrule-namespace-and-wcfrelay/azuredeploy.parameters.json

```diff
@@ -0,0 +1,21 @@
+{
+    "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "namespaceName": {
+            "value": "GEN-UNIQUE-RL-NM-001"
```

Comment:
None of the GEN values here are valid, if they don't need to be globally unique, just use a defaultValue in the template - if they do need to be unique then use GEN-UNIQUE

Reply:
used GEN-UNOUE